lib/matplotlib/backend_bases.py

```diff
@@ -1428,6 +1428,17 @@ class DrawEvent(Event):
     In addition to the :class:`Event` attributes, the following event
     attributes are defined:
```

Comment:
I think this sentence needs to get moved down now?

Reply:
Done, but the diff resolved with out this line changing.